patches/@docusaurus+mdx-loader+2.4.3.patch

```diff
@@ -7,9 +7,12 @@ index 5f9dce1..fdc1976 100644
      const reqOptions = this.getOptions();
      const { frontMatter, content: contentWithTitle } = (0, utils_1.parseFrontMatter)(fileString);
 +
-+    const ogImageURL = new URL('https://apify.com/og-image/docs-article');
-+    ogImageURL.searchParams.set('title', frontMatter.title);
-+    frontMatter.image ??= ogImageURL.toString();
++    const isPartial = filePath.split('/').pop()[0] === '_';
++    if (!isPartial) {
```

Comment:
FYI @barjin and @B4nan I'm modifying this delicate piece of code so that it doesn't affect partials. Here on GitHub it's a diff of a diff file, so it might be hard to read—I'm adding

```js
const isPartial = filePath.split('/').pop()[0] === '_';
```

and then

```js
if (!isPartial) {
  ...  // the same code which was already there
}
```

Without my change Docusaurus brings up a warning that we shouldn't use front matter in partials. I tested the change manually and `isPartial` is correctly `true` for `_exercises.mdx` and `false` for everything else.

Reply:
Good thinking, we shouldn't need this anymore anyway, this is a perfect use case for a [custom frontmatter parser](https://github.com/facebook/docusaurus/pull/9624) once we update to Docusaurus v3.